java/app/src/main/java/com/fullstorydev/shoppedemo/MainActivity.java

```diff
@@ -69,4 +75,9 @@ private void hideKeyboard(View view) {
             imm.hideSoftInputFromWindow(view.getWindowToken(), InputMethodManager.HIDE_NOT_ALWAYS);
         }
     }
+
+    @Override
+    public void onReady(FSSessionData sessionData) {
+        Log.d("MainActivity", "FS session URL: " + FS.getCurrentSessionURL());
```

Comment:
This _should_ get blocked by the default log levels, but I would hesitate to print out a url like this, since outside people won't be able to access it.

Reply:
Maybe add a comment about how to change the log level - or convert this to a commented-out or debug-only println or something. Your call, but I would prefer not getting into the habit of public code printing out the session url, since that should be an internal/debug thing.

Reply:
Good shout! Removed logging and added a comment on retrieving the session url to send to any other system.